addon/components/au-file-upload.gts

```diff
@@ -302,6 +312,13 @@ function isValidExtension(
   return validExtensions.includes(extension);
 }
 
-function isValidFileSize(fileSize: number, maximumSize: number): boolean {
-  return fileSize < maximumSize * Math.pow(1024, 2);
+function isValidFileSize(
```

Comment:
This isn't used anymore, that's also what the linter is complaining about.

Reply:
oups that should have been git stashed, somehow got through. Good linter :+1: thumbs